src/pages/admin/AdminHomePage.module.scss

```
@@ -0,0 +1,110 @@
+.page {
+  display: flex;
+  flex-direction: column;
+  gap: 1.25rem;
+}
+
+.stats {
+  display: grid;
+  gap: 1rem;
+  // 좁아지면 4 → 2 → 1 열로 자연히 접힌다. 브레이크포인트를 따로 두지 않는다.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)